src/ucp/rma/rma_send.c

```diff
@@ -268,6 +268,7 @@ ucs_status_ptr_t ucp_put_nbx(ucp_ep_h ep, const void *buffer, size_t count,
         req->send.rma.rkey        = rkey;
         req->send.rma.remote_addr = remote_addr;
 
+        ucp_request_send_state_clear(req, NULL);
```

Comment:
no need to do it under "proto_enable" - it will have own abort function which does not assume comp_cb is initialized

Reply:
it failed at least once in CI